packages/cli/src/services/jwt.service.ts

```diff
@@ -4,14 +4,12 @@ import config from '@/config';
 
 @Service()
 export class JwtService {
-	private readonly userManagementSecret = config.getEnv('userManagement.jwtSecret');
-
 	public signData(payload: object, options: jwt.SignOptions = {}): string {
-		return jwt.sign(payload, this.userManagementSecret, options);
+		return jwt.sign(payload, config.getEnv('userManagement.jwtSecret'), options);
```

Comment:
this had to made because of a bug in dev mode that [this](https://github.com/n8n-io/n8n/pull/7550) PR fixes